modules/claims_api/app/swagger/claims_api/v0/form_526_controller_swagger.rb

```diff
@@ -521,6 +521,7 @@ class Form526ControllerSwagger
 
           response 200 do
             key :description, 'upload response'
+            # TODO is there anything that should be shown as an example response body?
```

Comment:
the example response is empty, not sure if that's an oversight

Reply:
It is an oversight and it looks like no one ever completed the TODO, if you hit the upload end point you get back the ClaimsShow schema with additional files listed. https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb#L47

Reply:
no worries, I added that TODO as I was going through, so it wasn't previously there, just in my audit

- [ ] Add `ClaimsShow` schema as example response